hooks/_common.sh

```diff
@@ -197,14 +197,16 @@ function common::is_hook_run_on_whole_repo {
 function common::get_cpu_num {
   local -r parallelism_ci_cpu_cores=$1
 
+  local cpu_quota cpu_period cpu_num
   local millicpu
 
   if [[ -f /sys/fs/cgroup/cpu/cpu.cfs_quota_us &&
     ! -f /proc/sys/fs/binfmt_misc/WSLInterop ]]; then # WSL have cfs_quota_us, but WSL should be checked as usual Linux host
     # Inside K8s pod or DinD in K8s
-    millicpu=$(< /sys/fs/cgroup/cpu/cpu.cfs_quota_us)
+    cpu_quota=$(< /sys/fs/cgroup/cpu/cpu.cfs_quota_us)
+    cpu_period=$(cat /sys/fs/cgroup/cpu/cpu.cfs_period_us || echo "$cpu_quota")
```

Comment:
It makes sense to suppress error
```suggestion
    cpu_period=$(cat /sys/fs/cgroup/cpu/cpu.cfs_period_us 2> /dev/null || echo "$cpu_quota")
```